.github/workflows/js-tests.yml

```diff
@@ -63,7 +63,6 @@ jobs:
 
     - name: Install Required Python Dependencies
       run: |
-        pip install -r requirements/pip.txt
```

Comment:
All of these removals are just to take advantage of the Make targets now doing this automatically.